src/Plugin.tsx

```diff
@@ -75,6 +84,14 @@ const PluginInner = (propsFromParent: EnrollmentOverviewProps) => {
         );
     }
 
+    if (isErrorTei || isErrorEvents) {
+        return (
+            <GenericError
+                errorMessage={i18n.t('Failed to load data. Please check that you have selected the correct program stage ID in the configuration.')}
+            />
```

Comment:
Write either "programStageId" or "program stage ID", add "" to the current string